rhel8/CMakeLists.txt

```diff
@@ -9,7 +9,6 @@ set(DISA_SRG_TYPE "os")
 ssg_build_product(${PRODUCT})
 
 ssg_build_html_table_by_ref(${PRODUCT} "nist")
-ssg_build_html_table_by_ref(${PRODUCT} "cis")
```

Comment:
Can the CIS table still be built, but not have the C2S profile?

e.g. many rules in other profiles (CUI, STIG) have CIS mappings.

Reply:
@shawndwells yes. fixed.